backend/calc.py

```diff
@@ -164,12 +234,21 @@ def crit_buffs(cls):
     def dh_buffs(cls):
         return {cls.BV, cls.BARD_DH, cls.DEVILMENT}
 
-    def avg_buff_effect(self):
-        return self.multiplier * self.duration / self.cd
+    def avg_buff_effect(self) -> float:
+        """Gives the dps expected from a buff"""
+        return self.multiplier * self.duration_sec / self.cooldown_sec
 
 
 class Jobs(Enum):
-    # job modifiers from https://www.akhmorning.com/allagan-studies/modifiers/
+    """
+    Contains job related info.
+
+    job_mod: The bonus given to the main stat.
```

Comment:
is this right? the tank classes in this enum have job mods based on vit and not str, just wanna confirm

Reply:
you're right, the code is wrong.

should use str mods:  105 for WAR and DRK, 100 for GNB and PLD

(wtf square?)

Reply:
also probably add a comment about tanks using str for damage

Reply:
done